portfolio/src/main/java/com/google/sps/data/AuthSend.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/* A (login or logout) url + bool isLoggedIn 
+ * object to send in json in AuthServlet.doGet
+ */
```

Comment:
Please try to make comments without spilling implementation details. Something like the following (not sure if I described it 100% correct, but the idea is that I don't mention isLoggedIn, json, and other implementation details that may change in the future)

Class AuthSend represents an object that is used in the authentication servlet
to allow login and logout actions.